lib/logflare_web/live/search_live/logs_search_lv.ex

```diff
@@ -887,4 +891,31 @@ defmodule LogflareWeb.Source.SearchLV do
     |> assign(:querystring, qs)
     |> assign(:lql_rules, lql_rules)
   end
+
+  defp check_suggested_fields(_lql_rules, _source, %{assigns: %{force_query: true}} = socket) do
+    {:ok, assign(socket, :force_query, false)}
+  end
+
+  defp check_suggested_fields(lql_rules, %{suggested_fields: ""}, socket) do
+    {:ok, socket}
+  end
+
+  defp check_suggested_fields(
+         lql_rules,
+         %{suggested_fields: suggested_fields},
+         %{assigns: %{force_query: false}} = socket
+       ) do
+    suggested_present =
+      suggested_fields
+      |> String.split(",")
+      |> Enum.map(fn
+        "m." <> suggested_field -> "metadata." <> suggested_field
+        suggested_field -> suggested_field
+      end)
+      |> Enum.all?(fn suggested_field ->
+        Enum.find(lql_rules, fn %{path: path} -> path == suggested_field end)
+      end)
+
+    if suggested_present, do: {:ok, socket}, else: {:error, :suggested_field_not_found}
```

Comment:
So even if the user acknowledges it, they will still receive the pop-up? Might get noisy (and irritating) and overlap with other flashes. I think having a one time flag to show it once only will be good

Reply:
Will change